.github/workflows/README.md

```diff
@@ -0,0 +1,49 @@
+# GitHub Actions CI
+
+## Actions
+ - `custom-actions/build-packager`:
+   Builds Shaka Packager.  Leaves build artifacts in the "artifacts" folder.
+   Requires OS-dependent and build-dependent inputs.
+ - `custom-actions/test-packager`:
+   Tests Shaka Packager.  Requires OS-dependent and build-dependent inputs.
+ - `custom-actions/build-docs`:
+   Builds Shaka Packager docs.
+
+## Workflows
+ - On PR:
+   - `build_and_test.yaml`:
+     Builds and tests all combinations of OS & build settings.  Also builds
+     docs.
+ - On release tag:
+   - `github_release.yaml`:
+     Creates a draft release on GitHub, builds and tests all combinations of OS
+     & build settings, builds docs on all OSes, attaches static release binaries
+     to the draft release, then fully publishes the release.
+ - On release published:
+   - `docker_hub_release.yaml`:
+     Builds a Docker image to match the published GitHub release, then pushes it
+     to Docker Hub.
```

Comment:
It will be useful to publish the latest ToT image to Docker Hub too with :latest tag on all commits. We used to do that but it was stopped due to permission issues about 2 years ago. With GitHub actions, it seems like we can do it again.

Reply:
Absolutely.  Right now, `:latest` is being set on the new release images.